java/examples/postgresql.java

```diff
@@ -0,0 +1,16 @@
+package sh.platform.config.sample;
+
+import sh.platform.config.Config;
+import sh.platform.config.PostgreSQL;
+
+import javax.sql.DataSource;
+
+public class PostgreSQLSample {
+
+    public static void main(String[] args) {
+        Config config = new Config();
+        PostgreSQL database = config.getCredential("database", PostgreSQL::new);
+        DataSource dataSource = database.get();
```

Comment:
These files should also show working with the services, not just opening the connection.  That serves two purposes:

1) Provides documentation for customers on how to talk to every service, at least at a basic level.
2) It acts as a canary test for us to verify that every language can talk to every service.  If that breaks we know there's a problem somewhere, before it hits a user.

Reply:
See the pattern in the other language examples, which I believe are all fairly consistent.

Reply:
unlike python, PHP, I need to convert to bytecode before executing the code.
The whole idea is to run the command while it is showing?